### PR TITLE
Remove deprecated benchmark scripts

### PR DESCRIPTION
**Pull request by portante**
Documentation changes for PR #2545.

### CODE REVIEW

File: doc.html [hunk 1]
```diff
@@ -1,1005 +1,1003 @@
 <!DOCTYPE html>
 <html>
+    <head>
+        <meta charset="utf-8">
+        <title></title>
+        <meta name="author" content="">
+        <meta name="description" content="">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link href="css/style.css" rel="stylesheet">
+        <link href="css/fontawesome.min.css">
```
Comment (dbutenhof) [inline doc.html:10]: So, not *quite* just massive whitespace changes in the first commit. 😆

**Review by dbutenhof — APPROVED** [review]
Do we want to remove all mention entirely, or would it maybe be better to add them in a *list* of deprecated scripts that will be removed. It wouldn't need any additional details, since obviously we don't want to encourage anyone to use them ... but it might be more noticeable, to anyone who bothers to skim the document, that "something is happening", than simply having them drop out.

File: doc.html (hunk 1 above)
Comment (portante) [reply doc.html:10]: Refactored into a separate commit, thanks for pointing out!

**Comment by portante** [comment]
> Do we want to remove all mention entirely ...

I don't think it is worth keeping mention of the deprecated scripts in there because there is 0 evidence of use for years, the `pbench-tpcc` does not actually exist, and we are missing `pbench-cyclictest`, `pbench-iozone`, and `pbench-netperf`.

Best to let sleeping dogs lie.

**Pull request merged**